spaces/domains.json

```
@@ -27,4 +27,5 @@
   "voting.pancakeswap.finance": "pancake",
   "vote.taconomics.io": "taco",
   "gov.narwhalswap.org": "narwhal"
+  "governance.etgproject.org": "etgfinance"
```

Please commit only the domains.json file. You've committed all your space again.
Add a comma at the end of the line above your domain.
```suggestion
  "gov.narwhalswap.org": "narwhal",
  "governance.etgproject.org": "etgfinance"
```